### DISCUSSION TITLE
Software dimming

### DISCUSSION THREAD

**Discussion by tgeorgiop** [MonitorControl/MonitorControl#986] [Q&A] [answered]
Hello, is software dimming like pwm flickering? How the monitor dims? Is something like flickering being applied?

**Comment by waydabber** [comment, marked as answer]
Hey @tgeorgiop 

No, software dimming does not influence the display backlight in any way. There are two methods in the app:

- Gamma table control - it tones down the luminance range of the screen. Whites will be gray, grays darker gray etc..
- Shade based dimming - effectively puts a large transparent black screen over the entire desktop - the transparency is reduced as dimming increases.

Hardware control (like DDC) affects the backlight LEDs of the display itself - whether it is PWM or not is up to the display hardware.

Hope this helps!
